### PR TITLE
Fix python code description

### LINKED ISSUE #772: [Japanese/Python] WorkshopPipeline fails to execute
### Describe the bug

WorkshopPipelineStack (CodePipeline) execution fails.

### Expected Behavior

WorkshopPipelineStack runs successfully.

### Current Behavior

The following error occurred during the CodeBuild.

```
[Container] 2022/11/16 07:56:02 Running command cdk synth
Traceback (most recent call last):
  File "/codebuild/output/src486291841/src/app.py", line 6, in <module>
    from cdk_workshop.pipeline_stack import WorkshopPipelineStack
  File "/codebuild/output/src486291841/src/cdk_workshop/pipeline_stack.py", line 8, in <module>
    from pipeline_stage import WorkshopPipelineStage
ModuleNotFoundError: No module named 'pipeline_stage'

Subprocess exited with error 1
```

### Reproduction Steps

This occurs when the Japanese version of the procedure is performed.
It does not occur in the English version.

### Possible Solution

Modify the following to match the English version.

```diff
  from constructs import Construct
  from aws_cdk import (
      Stack,
      aws_codecommit as codecommit,
      pipelines as pipelines,
  )
- from pipeline_stage import WorkshopPipelineStage
+ from cdk_workshop.pipeline_stage import WorkshopPipelineStage

  class WorkshopPipelineStack(Stack):
```

### Additional Information/Context

_No response_

### CDK CLI Version

2.50.0 (build 4c11af6)

### Section

Advanced Topics / CDK Pipelines / Add Application to Pipeline (Japanese)

### Browser

Chrome

### Language

Python

### PR DESCRIPTION
**Pull request by 3socha**
Fixes #772

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT-0 License].

[MIT-0 License]: https://github.com/aws/mit-0/blob/master/MIT-0

### CODE REVIEW

File: workshop/content/english/30-python/70-advanced-topics/200-pipelines/3000-new-pipeline.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Now we are ready to define the basics of the pipeline.
 
 Return to the file `pipeline_stack.py` and edit as follows:
 
-{{<highlight python "hl_lines=13 18-30">}}
+{{<highlight python "hl_lines=12 17-29">}}
```
Comment (peterwoodworth) [inline workshop/content/english/30-python/70-advanced-topics/200-pipelines/3000-new-pipeline.md:12]: The differences I see are on lines `5 17-29` aside from the differences in formatting on lines 13-16. Where are you getting 12 from exactly?

Additionally when reviewing this, I noticed on the [previous page](https://cdkworkshop.com/30-python/70-advanced-topics/200-pipelines/2000-create-repo.html) that there's a dangling comma after the codecommit import line, as well as the dangling comma on _this_ page after the pipelines import in both the english and JA workshops

File: workshop/content/japanese/30-python/50-table-viewer/300-add.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ weight = 300
 
 ## Table viewer をスタックに追加
 
-`TableViewer` をスタックに追加するには、`cdk_workshop_stack.py` に次のコードでハイライトされている部分を追加する。
+`TableViewer` をスタックに追加するには、`cdk_workshop_stack.py` に次のコードでハイライトされている部分を追加します。
```
Comment (peterwoodworth) [inline workshop/content/japanese/30-python/50-table-viewer/300-add.md:8]: Is this a grammatical change? Translation service returns the same string

File: workshop/content/japanese/30-python/70-advanced-topics/200-pipelines/3000-new-pipeline.md [hunk 3]
```diff
@@ -9,14 +9,13 @@ weight = 130
 
 `pipeline_stack.py` を開き、以下のように編集します。
 
-{{<highlight python "hl_lines=13 18-30">}}
+{{<highlight python "hl_lines=12 17-29">}}
```
Comment (peterwoodworth) [inline workshop/content/japanese/30-python/70-advanced-topics/200-pipelines/3000-new-pipeline.md:12]: Same as english

**Review by peterwoodworth — CHANGES_REQUESTED** [review]
Thanks for this! Just a few comments to address

File: workshop/content/english/30-python/70-advanced-topics/200-pipelines/3000-new-pipeline.md (hunk 1 above)
Comment (3socha) [reply workshop/content/english/30-python/70-advanced-topics/200-pipelines/3000-new-pipeline.md]: Oops, it seems I made a mistake in that part.
As you mentioned, the differences between the previous page and this one are indeed on lines 5 and 17-29 for both the English and Japanese versions.
I will correct the 12 to 5.

File: workshop/content/japanese/30-python/70-advanced-topics/200-pipelines/3000-new-pipeline.md (hunk 3 above)
Comment (3socha) [reply workshop/content/japanese/30-python/70-advanced-topics/200-pipelines/3000-new-pipeline.md]: Fixed the same as the English version.

File: workshop/content/japanese/30-python/50-table-viewer/300-add.md (hunk 2 above)
Comment (peterwoodworth) [reply workshop/content/japanese/30-python/50-table-viewer/300-add.md:8]: Thanks for the clarification!

**Pull request merged**